### PR TITLE
allow activity logger to have workflow id and run id

### PR DESCRIPTION
**Pull request by lihannan99**
This should fix https://github.com/uber-go/cadence-client/issues/637

### CODE REVIEW

File: internal/activity.go [hunk 1]
```diff
@@ -269,6 +270,13 @@ func WithActivityTask(
 	} else {
 		deadline = startToCloseDeadline
 	}
+
+	logger.With(
+		zapcore.Field{Key: tagWorkflowType, Type: zapcore.StringType, String: *task.WorkflowType.Name},
+		zapcore.Field{Key: tagWorkflowID, Type: zapcore.StringType, String: *task.WorkflowExecution.WorkflowId},
+		zapcore.Field{Key: tagRunID, Type: zapcore.StringType, String: *task.WorkflowExecution.RunId},
+	)
```
Comment (mfateev) [inline internal/activity.go]: Does it already include activity type?

Comment (lihannan99) [reply internal/activity.go]: Good point, I will add that as well

Comment (lihannan99) [reply internal/activity.go]: Do we also want to add activity ID ?

Comment (lihannan99) [reply internal/activity.go]: I added activity ID and activity type to it.

Comment (mfateev) [reply internal/activity.go]: Is ActivityID ok? As the number of them is potentially unbounded.

**Pull request merged**